File: app/src/main/java/com/tda/app/data/model/response/CartItemResponse.java
package com.tda.app.data.model.response;

import java.util.List;


public class CartItemResponse {
    public long itemId;
    public int quantity;
    public double amount;
    //product
    public String productCode;
    public String productName;
    public int status;
    public double price;
    public double promotionPrice;
    public long categoryId;
    public List<String> images_file;




}

File: app/src/main/java/com/tda/app/model/response/CartItemResponse.kt
package com.tda.app.model.response

data class CartItemResponse(
    val itemId: Long,
    val quantity: Int,
    val amount: Double,

    //product
    val productCode: String,
    val productName: String,
    val status: Int,
    val price: Double,
    val promotionPrice: Double,
    val categoryId: Long,
    val images_file: List<String>
)